client/signup/steps/plans/index.jsx

```diff
@@ -141,6 +140,7 @@ module.exports = React.createClass( {
 		return <PlansCompare
 			className="plans-step__compare"
 			enableFreeTrials={ this.isFreeTrialFlow() }
+			flowName={ this.props.flowName }
```

Comment:
Instead of passing `flowName` here, could we check the `enableFreeTrials` prop in `PlansCompare` instead?